Ensure native byte order for memmap. (#1354)
joblib/joblib: CHANGES.rst

In development
--------------

> - Ensure native byte order for memmap arrays when pickling.

Reviewer: ```suggestion
- Ensure native byte order for memmap arrays in `joblib.load`.
```
